tests/topotests/mgmt_config/r1/mgmtd.conf

```diff
@@ -0,0 +1,11 @@
+debug northbound notifications
```

Comment:
I think we decided to pull off debug stuff from CI?

Reply:
The test is actually saving the debug log for each config file load into files named after the config file being loaded (see save_log_snippet). This is very (the only) useful information when trying to figure out why something went wrong. :)